packages/xarc-app/src/index.ts

```diff
@@ -297,7 +299,7 @@ if (!getAppMode().hasEnv()) {
   const guessAppSrcDir = () => {
     if (module.parent && module.parent.filename) {
       const fn = module.parent.filename;
-      const dir = fn.substr(process.cwd().length + 1).split("/")[0];
+      const dir = fn.substr(detectXARCPath(archetype.options.cwd)).split("/")[0];
```

Comment:
the new code is wrong, look at the old code.

Reply:
Fixed going to check in